allensdk/brain_observatory/vbn_2022/metadata_writer/metadata_writer.py

```diff
@@ -1,278 +1,281 @@
-import argschema
 import pathlib
 import time
-import pandas as pd
 
 import allensdk
-
-from allensdk.core.dataframe_utils import (
-    patch_df_from_other)
-
-from allensdk.brain_observatory.vbn_2022.metadata_writer.schemas import (
-    VBN2022MetadataWriterInputSchema,
-    DataReleaseToolsInputSchema)
-
-from allensdk.brain_observatory.data_release_utils \
-    .metadata_utils.id_generator import (
-        FileIDGenerator)
-
-from allensdk.brain_observatory.data_release_utils \
-    .metadata_utils.utils import (
-        add_file_paths_to_metadata_table)
-
+import argschema
+import pandas as pd
+from allensdk.brain_observatory.data_release_utils.metadata_utils.id_generator import ( # noqa
```

Comment:
Did you need to manually add the # noqa? 
The isort script only adds it to the end of the import statement instead of the end of the first line of a multiline import statement.